File: 1817-finding-the-users-active-minutes/1817-finding-the-users-active-minutes.java
class Solution {
    public int[] findingUsersActiveMinutes(int[][] logs, int k) {
        Map<Integer, Set<Integer>> map = new HashMap<>();
        for(int i=0;i<logs.length;i++) {
            map.putIfAbsent(logs[i][0], new HashSet<>());
            map.get(logs[i][0]).add(logs[i][1]);
        }
        int[] res=new int[k];
        for(int key : map.keySet()) 
            res[map.get(key).size()-1]++;
        return res;
    }
}

File: 1817-finding-the-users-active-minutes/1817-finding-the-users-active-minutes.cpp
class Solution {
public:
    vector<int> findingUsersActiveMinutes(vector<vector<int>>& logs, int k) {
        unordered_map<int, unordered_set<int>> m;
    vector<int> res(k);
    for (auto &l : logs)
        m[l[0]].insert(l[1]);
    for (auto &p : m)
        ++res[p.second.size() - 1];
    return res;
    }
};